src/components/ChatTile.vue

```diff
@@ -1,8 +1,11 @@
 <template>
   <div :class="chatIsOpen ? 'chat-wrapper open' : 'chat-wrapper'">
     <div>
-      <button class="chat" @click="handleChatClick">
-        <template v-if="chatIsOpen">
+      <button class="chat" @click="handleChatClick" :disabled="!model">
```

Comment:
Here is where the main logic is to account for the potential timing issue of model load, basically disabling chat until the model exists.